src/hl7v2/schema.clj

```diff
@@ -50,21 +50,25 @@
          (drop (if (node-attrs node) 2 1))
          (into []))))
 
-(defn node-annotation [node]
+(defn en-doc-kw [node]
   (letfn [(rm-diacritics [s]
             (when s
               (-> s
                   (Normalizer/normalize Normalizer$Form/NFD)
                   (str/replace #"[\u0300-\u036f]" ""))))]
     (->> (node-children node)
          (some #(when (and (= "documentation" (first %))
-                           (= "en" (:lang (second %))))
-                  (->> (str/split (last %) #"\s+")
+                           (= (:lang @config) (:lang (second %))))
```

Comment:
Why do you use `@config` instead only `config`. Is it because there is `delay` into config body?

Reply:
yes, in clojure this is known as deref or dereferencing, you will need to get used to this, because there are many scenarios for deref in clojure, promise, delay, atoms